src/ensembl/src/gql-client.ts

```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
-import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client';
+import fetch from 'cross-fetch';
 
 export const client = new ApolloClient({
-  uri: '/api/thoas',
+  link: new HttpLink({ uri: 'api/thoas', fetch }),
```

Comment:
why was this necessary?

Reply:
This is because Node.js doesn't have fetch and we need to pass cross-fetch to make it available for it during SSR